torch/csrc/jit/passes/onnx/prepare_inplace_ops_for_onnx.cpp

```diff
@@ -27,9 +27,10 @@ Value* ConvertSliceToIndex(Node* slice, Value* size, Node* insertBefore) {
   const int64_t int_max = std::numeric_limits<int>::max();
   auto graph = slice->owningGraph();
   WithInsertPoint guard(insertBefore);
-  auto start = slice->get(attr::start) ? slice->get(attr::start) : 0;
-  auto end = slice->get(attr::end) ? slice->get(attr::end) : int_max;
-  auto step = slice->get(attr::step);
+  TORCH_INTERNAL_ASSERT((slice->inputs()).size() == 5);
```

Comment:
How about the opset < 11, if we set start and end as arguments, not inputs

Reply:
For this pass the operator is aten::slice, start/end/step would always be inputs. The function
ConvertSliceToIndex is only called on encountering an index_put node.